client/src/components/content/BpmnModeler/BpmnModeler.jsx

```diff
@@ -59,25 +59,11 @@ const BpmnModeler = () => {
    * @description Will parse a given xml file into a .robot file and download it
    * @param {string} xml String that sets the xml to be parsed
    */
-  const downloadRobotFile = (xml) => {
-    const body = convert.xml2json(xml, { compact: true, spaces: 4 });
-    const jsonBody = JSON.parse(body);
-    const robot = parseDiagramJson(jsonBody);
-    downloadString(robot, 'text/robot', 'testRobot.robot');
-  };
-
-  /**
-   * @description Will get the underlying xml of the current bpmn diagram, parse it into a .robot file and download it.
-   */
-  const getRobotFile = () => {
-    modeler
-      .saveXML()
-      .then((json) => {
-        const { xml } = json;
-        downloadRobotFile(xml);
-      })
-      .catch((error) => {
-        console.error(error);
+  const downloadRobotFile = () => {
+    getParsedRobotFile()
```

Comment:
Where is this defined?

Reply:
I think it is the interface to be called when the parsers are conencted

Reply:
good point. It seems like somehow the import got lost. I added the import again

Reply:
i did that already :D